turbo.json

```diff
@@ -11,21 +11,28 @@
       "outputs": ["css/**"]
     },
     "@itwin/itwinui-react#build": {
-      "dependsOn": ["^build"],
+      "dependsOn": ["^build", "@itwin/itwinui-css#build"],
       "outputs": ["esm/**", "cjs/**"]
     },
+    "css-workshop#build": {
+      "dependsOn": ["@itwin/itwinui-css#build"],
+      "outputs": ["dist/**", "build/**"]
+    },
     "react-workshop#build": {
       "dependsOn": ["^build"],
       "outputs": ["build/**"]
     },
+    "portal": {
```

Comment:
Is this supposed to be `portal#build`? Because I have occasionally had some build failures after this PR was merged, and I think this might be the culprit.

Reply:
Whoops yes. For now you can manually run dependent build commands first.

Also would be useful if you review the PR next time 😅